2020-12-14 14:53:36,283 DEBUG - (nikko.aratan) - Successfully loaded and updated table
2020-12-14 14:55:11,707 DEBUG - (rock.lim) - Successfully loaded and updated table
2020-12-14 16:32:50,230 DEBUG - starting thread (client mode): 0x48f4ae90L
2020-12-14 16:32:50,231 DEBUG - Local version/idstring: SSH-2.0-paramiko_2.6.0
2020-12-14 16:33:05,233 ERROR - Exception: Error reading SSH protocol banner
2020-12-14 16:33:05,235 ERROR - Traceback (most recent call last):
2020-12-14 16:33:05,235 ERROR -   File "/app/argus_final/venv/lib/python2.7/site-packages/paramiko/transport.py", line 2039, in run
2020-12-14 16:33:05,235 ERROR -     self._check_banner()
2020-12-14 16:33:05,235 ERROR -   File "/app/argus_final/venv/lib/python2.7/site-packages/paramiko/transport.py", line 2216, in _check_banner
2020-12-14 16:33:05,235 ERROR -     "Error reading SSH protocol banner" + str(e)
2020-12-14 16:33:05,235 ERROR - SSHException: Error reading SSH protocol banner
2020-12-14 16:33:05,235 ERROR - 
2020-12-14 22:53:49,500 ERROR - Internal Server Error: /proxycontroller/
Traceback (most recent call last):
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/contrib/auth/decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/app/argus_final/app/argus/argus/mixins.py", line 14, in dispatch
    return super(GENERIC_PERMISSION_MIXIN, self).dispatch(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/app/argus_final/app/argus/proxycontroller/views.py", line 241, in get
    context['connected'][channel['celery_queue']] = REDIS.get_data('SOCKET_MONITOR-CONNECTED-{0}'.format(channel['celery_queue']),index=4)
  File "/app/argus_final/app/argus/global_objects/REDIS_DATA_MANAGER.py", line 25, in get_data
    return json.loads(self.__redishost[index].get(key))
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
TypeError: expected string or buffer
2020-12-15 10:17:06,232 DEBUG - (nikko.aratan) - Successfully loaded and updated table
2020-12-15 10:17:38,603 DEBUG - (nikko.aratan) - Successfully loaded and updated table
2020-12-15 10:18:07,573 DEBUG - (nikko.aratan) - Successfully loaded and updated table
2020-12-15 10:18:11,333 DEBUG - (nikko.aratan) - Successfully loaded and updated table
2020-12-15 10:18:14,948 DEBUG - (nikko.aratan) - Successfully loaded and updated table
2020-12-15 10:18:27,176 DEBUG - (nikko.aratan) - Successfully loaded and updated table
2020-12-15 10:23:08,517 DEBUG - (nikko.aratan) - Successfully loaded and updated table
2020-12-15 10:23:29,813 DEBUG - (nikko.aratan) - Successfully loaded and updated table
2020-12-15 10:34:37,251 DEBUG - (nikko.aratan) - Successfully loaded and updated table
2020-12-15 10:35:53,486 DEBUG - (nikko.aratan) - Successfully loaded and updated table
2020-12-15 10:41:49,649 DEBUG - (jerry.lam) - Successfully loaded and updated table
2020-12-15 12:03:32,925 DEBUG - (jerry.lam) - Successfully loaded and updated table
2020-12-15 15:27:06,759 DEBUG - (yroll.macalino) - Successfully loaded and updated table
2020-12-15 16:39:10,605 DEBUG - starting thread (client mode): 0x49972b90L
2020-12-15 16:39:10,606 DEBUG - Local version/idstring: SSH-2.0-paramiko_2.6.0
2020-12-15 16:39:10,848 DEBUG - Remote version/idstring: SSH-2.0-OpenSSH_7.4
2020-12-15 16:39:10,848 INFO - Connected (version 2.0, client OpenSSH_7.4)
2020-12-15 16:39:11,042 DEBUG - kex algos:[u'curve25519-sha256', u'curve25519-sha256@libssh.org', u'ecdh-sha2-nistp256', u'ecdh-sha2-nistp384', u'ecdh-sha2-nistp521', u'diffie-hellman-group-exchange-sha256', u'diffie-hellman-group16-sha512', u'diffie-hellman-group18-sha512', u'diffie-hellman-group-exchange-sha1', u'diffie-hellman-group14-sha256', u'diffie-hellman-group14-sha1', u'diffie-hellman-group1-sha1'] server key:[u'ssh-rsa', u'rsa-sha2-512', u'rsa-sha2-256', u'ecdsa-sha2-nistp256', u'ssh-ed25519'] client encrypt:[u'chacha20-poly1305@openssh.com', u'aes128-ctr', u'aes192-ctr', u'aes256-ctr', u'aes128-gcm@openssh.com', u'aes256-gcm@openssh.com', u'aes128-cbc', u'aes192-cbc', u'aes256-cbc', u'blowfish-cbc', u'cast128-cbc', u'3des-cbc'] server encrypt:[u'chacha20-poly1305@openssh.com', u'aes128-ctr', u'aes192-ctr', u'aes256-ctr', u'aes128-gcm@openssh.com', u'aes256-gcm@openssh.com', u'aes128-cbc', u'aes192-cbc', u'aes256-cbc', u'blowfish-cbc', u'cast128-cbc', u'3des-cbc'] client mac:[u'umac-64-etm@openssh.com', u'umac-128-etm@openssh.com', u'hmac-sha2-256-etm@openssh.com', u'hmac-sha2-512-etm@openssh.com', u'hmac-sha1-etm@openssh.com', u'umac-64@openssh.com', u'umac-128@openssh.com', u'hmac-sha2-256', u'hmac-sha2-512', u'hmac-sha1'] server mac:[u'umac-64-etm@openssh.com', u'umac-128-etm@openssh.com', u'hmac-sha2-256-etm@openssh.com', u'hmac-sha2-512-etm@openssh.com', u'hmac-sha1-etm@openssh.com', u'umac-64@openssh.com', u'umac-128@openssh.com', u'hmac-sha2-256', u'hmac-sha2-512', u'hmac-sha1'] client compress:[u'none', u'zlib@openssh.com'] server compress:[u'none', u'zlib@openssh.com'] client lang:[u''] server lang:[u''] kex follows?False
2020-12-15 16:39:11,042 DEBUG - Kex agreed: curve25519-sha256@libssh.org
2020-12-15 16:39:11,043 DEBUG - HostKey agreed: ssh-ed25519
2020-12-15 16:39:11,043 DEBUG - Cipher agreed: aes128-ctr
2020-12-15 16:39:11,043 DEBUG - MAC agreed: hmac-sha2-256
2020-12-15 16:39:11,043 DEBUG - Compression agreed: none
2020-12-15 16:39:13,783 DEBUG - kex engine KexCurve25519 specified hash_algo <built-in function openssl_sha256>
2020-12-15 16:39:13,784 DEBUG - Switch to new keys ...
2020-12-15 16:39:13,809 DEBUG - Adding ssh-ed25519 host key for [103.100.209.23]:28032: fb679dcdfda01a6424c0563de3f822e1
2020-12-15 16:39:13,811 DEBUG - Trying discovered key f1e2a20412f7e99b7abdb041b22f15e1 in /root/.ssh/id_rsa
2020-12-15 16:39:14,291 DEBUG - userauth is OK
2020-12-15 16:39:22,449 INFO - Authentication (publickey) successful!
2020-12-15 16:39:22,462 DEBUG - [chan 0] Max packet in: 32768 bytes
2020-12-15 16:39:22,678 DEBUG - Received global request "hostkeys-00@openssh.com"
2020-12-15 16:39:22,678 DEBUG - Rejecting "hostkeys-00@openssh.com" global request from server.
2020-12-15 16:39:22,678 DEBUG - Debug msg: Adding to environment: NAME=argus
2020-12-15 16:39:22,943 DEBUG - [chan 0] Max packet out: 32768 bytes
2020-12-15 16:39:22,943 DEBUG - Secsh channel 0 opened.
2020-12-15 16:39:23,200 DEBUG - [chan 0] Sesch channel 0 request ok
2020-12-15 16:39:23,435 DEBUG - [chan 0] Sesch channel 0 request ok
2020-12-15 16:39:23,436 DEBUG - [chan 1] Max packet in: 32768 bytes
2020-12-15 16:39:23,667 DEBUG - [chan 1] Max packet out: 32768 bytes
2020-12-15 16:39:23,667 DEBUG - Secsh channel 1 opened.
2020-12-15 16:39:23,924 DEBUG - [chan 1] Sesch channel 1 request ok
2020-12-15 16:39:46,627 DEBUG - [chan 2] Max packet in: 32768 bytes
2020-12-15 16:39:46,640 DEBUG - [chan 1] EOF received (1)
2020-12-15 16:39:46,640 DEBUG - [chan 1] EOF sent (1)
2020-12-15 16:39:46,858 DEBUG - [chan 2] Max packet out: 32768 bytes
2020-12-15 16:39:46,859 DEBUG - Secsh channel 2 opened.
2020-12-15 16:39:47,364 DEBUG - [chan 2] Sesch channel 2 request ok
2020-12-15 16:39:47,389 DEBUG - [chan 2] EOF received (2)
2020-12-15 16:39:47,389 DEBUG - [chan 2] EOF sent (2)
2020-12-15 16:39:47,391 DEBUG - EOF in transport thread
2020-12-17 10:14:47,924 ERROR - Internal Server Error: /esync/om-fpms_pms_nginx/
Traceback (most recent call last):
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/contrib/auth/decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/app/argus_final/app/argus/argus/mixins.py", line 14, in dispatch
    return super(GENERIC_PERMISSION_MIXIN, self).dispatch(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/app/argus_final/app/argus/esync/views.py", line 169, in get
    context['keys'] = parse_standard_config(context['content'])
  File "/app/argus_final/app/argus/esync/lib/nginx_processor.py", line 11, in parse_standard_config
    rConfig = nginx.loads(config).as_dict
  File "/app/argus_final/venv/lib/python2.7/site-packages/nginx.py", line 402, in loads
    m = re.compile(r'^\s*events\s*{', re.S).search(data[index:])
TypeError: 'bool' object has no attribute '__getitem__'
2020-12-17 10:16:00,841 ERROR - Internal Server Error: /esync/om-fpms_pms_nginx/
Traceback (most recent call last):
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/contrib/auth/decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/app/argus_final/app/argus/argus/mixins.py", line 14, in dispatch
    return super(GENERIC_PERMISSION_MIXIN, self).dispatch(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/app/argus_final/app/argus/esync/views.py", line 169, in get
    context['keys'] = parse_standard_config(context['content'])
  File "/app/argus_final/app/argus/esync/lib/nginx_processor.py", line 11, in parse_standard_config
    rConfig = nginx.loads(config).as_dict
  File "/app/argus_final/venv/lib/python2.7/site-packages/nginx.py", line 402, in loads
    m = re.compile(r'^\s*events\s*{', re.S).search(data[index:])
TypeError: 'bool' object has no attribute '__getitem__'
2020-12-17 10:16:03,624 ERROR - Internal Server Error: /esync/om-fpms_pms_nginx/
Traceback (most recent call last):
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/contrib/auth/decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/app/argus_final/app/argus/argus/mixins.py", line 14, in dispatch
    return super(GENERIC_PERMISSION_MIXIN, self).dispatch(request, *args, **kwargs)
  File "/app/argus_final/venv/lib/python2.7/site-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/app/argus_final/app/argus/esync/views.py", line 169, in get
    context['keys'] = parse_standard_config(context['content'])
  File "/app/argus_final/app/argus/esync/lib/nginx_processor.py", line 11, in parse_standard_config
    rConfig = nginx.loads(config).as_dict
  File "/app/argus_final/venv/lib/python2.7/site-packages/nginx.py", line 402, in loads
    m = re.compile(r'^\s*events\s*{', re.S).search(data[index:])
TypeError: 'bool' object has no attribute '__getitem__'
